crates/sparrow-execution/src/tests.rs

```diff
@@ -0,0 +1,97 @@
+use arrow_array::cast::AsArray;
+use arrow_array::{Int64Array, StructArray, TimestampNanosecondArray, UInt64Array};
+use hashbrown::HashMap;
+use std::sync::Arc;
+
+use arrow_schema::{DataType, Field, Schema};
+use sparrow_arrow::{Batch, RowTime};
+
+#[test]
+fn test_logical_query() {
```

Comment:
Main test showing how it fits together -- create a tree from `sparrow-logical`, compile it, execute it.